MediaRecorder.html

```diff
@@ -495,7 +540,14 @@
             DOMException. If this paramater is not specified, the UA will use a
             platform-specific default format. The container format, whether
             passed in to the constructor or defaulted, will be used as the value
-            of the <code>mimeType</code> attribute.</dd>
+            of the <code>mimeType</code> attribute.
+
+            <div class="note"> <code><a>mimeType</a></code> specifies the media
+            type and container format for the recording via a type/subtype
+            combination, with the codecs and/or profiles parameters [[RFC6381]]
+            specified where ambiguity might arise. Individual codecs might have
+            further optional specific parameters. </div></dd>
```

Comment:
Might want to run this through a validator to see if there's not one `</dd>` too many now.